### PR TITLE
fix: Deploy successfully, by fixing circleci step FT-6941

### PR DESCRIPTION
**Pull request by kidsoncoffee**
This is part of https://codacy.atlassian.net/browse/FT-6941

### CODE REVIEW

**Comment by andreaTP** [comment]
CI is still red, can you check?

**Comment by kidsoncoffee** [comment]
@andreaTP I managed to run the test and found a bug in the *codacy-plugins-tests* repo. I opened a PR for that (https://github.com/codacy/codacy-plugins-test/pull/55). There is a discussion pending with **team orange** to understand if there are more changes due and how to tackle that, but with my change (that seems small) the fail is fixed.

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,20 +37,18 @@ jobs:
...
+          command: sbt -Dcodacy.tests.ignore.descriptions=true "runMain codacy.plugins.DockerTest json $CIRCLE_PROJECT_REPONAME:latest"
       - run:
           name: Test patterns
           working_directory: ~/codacy-plugins-test
-          command: sbt -Dcodacy.tests.noremove=true -Dcodacy.tests.threads=8 "run-main codacy.plugins.DockerTest pattern $CIRCLE_PROJECT_REPONAME:latest"
+          command: sbt -Dcodacy.tests.noremove=true -Dcodacy.tests.threads=8 "runMain codacy.plugins.DockerTest pattern $CIRCLE_PROJECT_REPONAME:latest"
       - deploy:
           name: Push application Docker image
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```
Comment (andreaTP) [inline .circleci/config.yml]: are we going to publish from every branch?

Comment (pedrocodacy) [reply .circleci/config.yml]: Please put this condition back, other than that also LGTM

Comment (kidsoncoffee) [reply .circleci/config.yml]: added

**Pull request merged**